docs/docs/features/database.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Database
+---
+
+# General characteristics
+
+**Juno** defines a database as an object that can store **key-value** pairs in a persistent way. Keys and values are
```

Comment:
I think it would be slightly easier to read if "Juno" always had a normal (non-bold) font face.